controllers/capabilities/openapi_controller.go

```diff
@@ -428,6 +428,17 @@ func (r *OpenAPIReconciler) validateOIDCSettingsInCR(openapiCR *capabilitiesv1be
 				}
 			}
 		}
+		// when OAS securitySchemes type is oauth2, and openapiCR spec is OIDC, then CR OIDC Authentication Flows parameters will be ignored,
```

Comment:
line 423 has ref to openapiref which causes constant updates to status of the CR which in turn re-reconciles on invalid error. Please double check the refs in invalidErrors. 
Thanks

Reply:
Updated. Thanks very much for this comment and discussion. Hope it's ok now. Images for 2 cases below:  
- No IssuerEndpoint nor IssuerEndpointRef found in OIDC spec in CR
![Screenshot from 2024-01-17 12-17-25](https://github.com/3scale/3scale-operator/assets/56625811/a4d7baf3-9b7f-4952-b138-3d778524c4c3)

- Missing OIDC definitions in CR
![Screenshot from 2024-01-17 12-22-23](https://github.com/3scale/3scale-operator/assets/56625811/063e65a4-68be-4e46-8e20-1ed1d860987a)